app/tests/test_models.py

```diff
@@ -1,7 +1,39 @@
+from flask import Flask
+from flask.ext.testing import TestCase
 from nose.tools import eq_
 
 from .. import models
 
+db = models.db
+
+class DbTestCase(TestCase):
+    BASE_APP_CONFIG = dict(
+        # TODO: We should use a test postgres db instead, since it's
+        # closer to our production deployment.
```

Comment:
Er yup, using postgres won't work until #23 is fixed: https://travis-ci.org/GovLab/noi2/builds/81077654

So for now I've added postgres support but **disabled** it until that issue is resolved.